ipyleaflet/leaflet.py

```diff
@@ -260,11 +260,24 @@ class Icon(UILayer):
 
     icon_url = Unicode('').tag(sync=True, o=True)
     shadow_url = Unicode(None, allow_none=True).tag(sync=True, o=True)
-    icon_size = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    shadow_size = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    icon_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    shadow_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
-    popup_anchor = List(None, default_value=[0, 0], allow_none=True, minlen=2, maxlen=2).tag(sync=True, o=True)
+    icon_size = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    shadow_size = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    icon_anchor = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    shadow_anchor = List(default_value=None, allow_none=True).tag(sync=True, o=True)
+    popup_anchor = List([0, 0], allow_none=True).tag(sync=True, o=True)
```

Comment:
Why is the default value not `None`?

Reply:
I am trying to respect the LeafletJS default values: https://leafletjs.com/reference-1.6.0.html#icon-popupanchor

Reply:
:+1:

Reply:
This makes me think I forgot to update them JS side #637